dashbuilder/dashbuilder-runtime/src/main/java/org/dashbuilder/client/screens/RuntimeScreen.java

```diff
@@ -67,6 +76,21 @@ public void loadDashboards(RuntimeModel runtimeModel) {
         NavTree navTree = runtimeModel.getNavTree();
         Menus menus = menusHelper.buildMenusFromNavTree(navTree).build();
         view.addMenus(menus);
+
+        goToIndex(runtimeModel.getLayoutTemplates());
```

Comment:
Can we move this call to the `RouterScreen.java`?
It looks like `loadDashboards` has more responsibilities than it implies.
So it makes more sense to me to let `RouterScreen` call `loadDashboards` and then `goToIndex`.

Reply:
done, thanks @caponetto